docs(concepts): fix link

hitsuji-no-shippo opened the pull request:
_describe your changes..._

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- __Remove these instructions from your PR as they are for your eyes only.__


[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/

hemal7735 commented on src/content/concepts/module-resolution.md:
```diff
@@ -55,18 +55,18 @@ import 'module';
-- If there is no `package.json` or if the main fields do not return a valid path, file names specified in the [`resolve.mainFiles`](/configuration/resolve/#resolve-mainfiles) configuration option are looked for in order, to see if a matching filename exists in the imported/required directory .
+- If the folder contains a `package.json` file, then fields specified in [`resolve.mainFields`](/configuration/resolve/#resolvemainfields) configuration option are looked up in order, and the first such field in `package.json` determines the file path.
+- If there is no `package.json` or if the main fields do not return a valid path, file names specified in the [`resolve.mainFiles`](/configuration/resolve/#resolvemainfiles) configuration option are looked for in order, to see if a matching filename exists in the imported/required directory .
 - The file extension is then resolved in a similar way using the `resolve.extensions` option.
```
can you add link for `resolve.extensions` ->  `/configuration/resolve/#resolveextensions`

hemal7735 commented:
hi @hitsuji-no-shippo , please sign the CLA to move forward this PR.

hitsuji-no-shippo commented:
I created this sentence using google translation because I can not speak English. I am sorry if this sentence is difficult to read.

@hemal7735 CLA's signature was invalidated because the commit e-mail address was different, so I fixed it. That's the reason for using force-pushed. I've also added a link.

montogeek commented:
@EugeneHlushko Why is this not getting caught in the build? Did we disable it?
cc @Munter

EugeneHlushko commented:
@montogeek this is not getting caught because the url responds with 200... Its the anchor that is unstable for some reason

Munter commented:
@montogeek @EugeneHlushko Hyperlink is supposed to check the correctness of fragment identifier links. Hyperlinks test suite has test cases for this exact situation. I can't explain why it doesn't discover these ones without doing a deeper debugging. My first intuition is that maybe this page is not on a discoverable part of the dependency graph, so none of its outgoing links are checked at all. But I haven't had time to spin up an environment and have a look at it yet

rishabh3112 commented:
@hitsuji-no-shippo there are some merge conflicts, rest lgtm :)

Munter commented:
@montogeek @EugeneHlushko Turns out there is indeed a bug in Hyperlink, which I have corrected on the current `master` branch. I'll make a release next time I have a bit of time and will start a branch to fix broken internal fragment links along with the update

Munter commented:
@montogeek 	@EugeneHlushko hyperlink@4.1.2 fixes the missing fragment check

hitsuji-no-shippo commented:
@rishabh3112 Thank you for telling me about conflicts. Just noticed today. I'm sorry for being late to notice that you said in July.
